### PR TITLE
Send for approval bug Fixed

### PR DESCRIPTION
**Pull request by LavishRawat**
Targets #2232

### Description
Send for approval bug Fixed

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/settings/hr/send-for-approval.blade.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<div class="card mt-4">
+	<form action="{{ url('/settings/hr/update') }}" method="POST">
+
+		{{ csrf_field() }}
```
Comment (P4NK4J) [inline resources/views/settings/hr/send-for-approval.blade.php:4]: What would happen if I use @csrf ?

**Review by P4NK4J — COMMENTED** [review]
Looks good, left a question

File: resources/views/settings/hr/send-for-approval.blade.php (hunk 1 above)
Comment (LavishRawat) [reply resources/views/settings/hr/send-for-approval.blade.php:4]: @P4NK4J  Nothing will change @csrf is the shorter syntax of {{ csrf_field( ) }}.

**Review by P4NK4J — APPROVED** [review]
Looks good

**Pull request merged**